.github/workflows/_build-wheel-release-upload.yml

```diff
@@ -22,47 +22,35 @@ on:
         required: true
 
 jobs:
-  tag-check:
-    runs-on: ubuntu-latest
-    outputs:
-      tag_valid: ${{ steps.tag-check.outputs.tag_valid }}
-    steps:
-      - name: Extract and validate tag for (pre)-release
-        id: tag-check
-        run: |
-          TAG_NAME=${GITHUB_REF#refs/tags/}
+  tag-privilege-check:
```

Comment:
Here, I've modularized checking the valid tag and privilege using a separate file `./.github/workflows/_release_tag_privilege_check.yml` that resides in this repository.

Reply:
OK, I like this